init.template.yaml

```diff
@@ -25,14 +28,17 @@ plans:
     in:
       - gitlabBackend
       - gitterDevChat
+    assignees:
```

Comment:
According to this comment https://github.com/microplan-xyz/microplan/issues/50#issuecomment-266495991 , gitlab supports only one assignee. Then why do we update this line ?